src/util/frame.js

```diff
@@ -9,5 +7,8 @@ import reqAnimationFrame from './request-animation-frame';
  * @return {func} The function wrapped within a requestAnimationFrame
  */
 export default function frame(func) {
-    return (...args) => reqAnimationFrame(() => func(...args));
+    console.warn(
+        'util.frame is deprecated; use window.requestAnimationFrame instead'
+    );
+    return (...args) => window.requestAnimationFrame(() => func(...args));
```

Comment:
no need to change the existing implementation.

Reply:
😂 we should keep the dep on a deprecated module? 😂

Reply:
What's up with the 😂 😂 ? I am trying to give you guidance and you think it's funny? Why the disrespect?

Reply:
Disrespect?! Not at all! =)

Reply:
And what does that even mean 'the dep on a deprecated module'? You can replace usage of util.frame in the code and simply leave util.frame alone, besides adding a deprecation warning.